DigitalLearningSolutions.Data.Tests/DataServices/LearningLogItemsDataServiceTests.cs

```diff
@@ -341,12 +355,18 @@ public void RemoveLearningLogItem_correctly_sets_removed_date_and_removed_by_id(
         private int InsertLearningLogItem(
             int delegateId,
             DateTime addedDate,
-            int competencyLearningResourceId
+            int learningResourceReferenceId
         )
         {
+            competencyLearningResourcesTestHelper.InsertLearningResourceReference(
+                learningResourceReferenceId,
+                1,
+                7,
+                "Resource"
+            );
             competencyLearningResourcesTestHelper.InsertCompetencyLearningResource(
                 1,
-                competencyLearningResourceId,
+                learningResourceReferenceId,
                 1,
                 7

```

Comment:
Side point that your previous fixes just got me to spot, the learningResourceReferenceId is technically in the CompetencyId parameter.